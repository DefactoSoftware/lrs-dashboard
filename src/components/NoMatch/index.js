import React from 'react';

export default ()=> <div>Nothing here</div>
