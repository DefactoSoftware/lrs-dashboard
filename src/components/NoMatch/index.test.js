import React from 'react';
import NoMatch from './index';
import renderer from 'react-test-renderer';

it('matches the render snapshot', () => {
  const tree = renderer.create(<NoMatch />).toJSON();

  expect(tree).toMatchSnapshot();
});
