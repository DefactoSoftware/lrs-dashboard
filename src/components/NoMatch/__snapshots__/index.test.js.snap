exports[`test matches the render snapshot 1`] = `
<div>
  Nothing here
</div>
`;
